cypress/e2e/filteringProducts.cy.ts

```diff
@@ -0,0 +1,30 @@
+import { CATEGORY } from "cypress/elements/category";
+import { FILTERS } from "cypress/elements/filters";
+import { NAVIGATION } from "cypress/elements/navigation";
+import { filterProducts, waitForProgressBarToNotBeVisible } from "cypress/support/shared";
+
+describe("Filter products", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should filter products by variant attribute SRS_0306", () => {
+    waitForProgressBarToNotBeVisible();
+    filterProducts(NAVIGATION.categoriesList, CATEGORY.categoryTitle);
+    cy.get(FILTERS.filtersMenu).first().click();
+    filterProducts(FILTERS.filterList, FILTERS.filterPill);
```

Comment:
Don't you want to check here if the graphql request for filtered products was sent and if all products from the response are displayed?

Reply:
I was thinking about that and it was little bit hard for me in this moment and we can do this if you would like to help me with that ;).